module/src/utils/dates.ts

```diff
@@ -0,0 +1,45 @@
+import { format, formatISO, parse, parseISO } from 'date-fns';
+import { enGB } from 'date-fns/locale';
+
+/** Set of utilities and types relating to the native JS date object */
+export namespace Dates {
+  /** The root type for date properties.
+   * - Accepts a date as a native JS `Date` object.
+   * - Accepts a date string in IOS format.
+   * - Accepts a date string in a format matching an accompanying `format` string property.
+   */
+  export type DateLike = Date | string;
```

Comment:
Should we support number as well for working with unix dates?

Reply:
I'll look into it.